README.md

```diff
@@ -270,20 +275,20 @@ For more examples of schema definitions, see the
 To serve the API spec from your application, first add the `OpenApiSpex.Plug.PutApiSpec` plug somewhere in the pipeline.
 
 ```elixir
-  pipeline :api do
-    plug OpenApiSpex.Plug.PutApiSpec, module: MyAppWeb.ApiSpec
-  end
+pipeline :api do
```

Comment:
Unless these code blocks aren't displaying properly in hex/github, then I'd prefer to leave them as-is.

Reply:
For consistency reason. All code block have no indentation except  these two. Let me know. :smile: